src/utils/image.ts

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-bitwise */
+import { imgType } from 'node-stego/es/helper'
+
+export function getDimension(buf: Uint8Array) {
+    const fallback = {
+        width: 0,
+        height: 0,
+    }
+    switch (imgType(buf)) {
+        case 'image/jpeg':
+            return getDimensionAsJPEG(buf) ?? fallback
+        case 'image/png':
+            return getDimensionAsPNG(buf)
+        default:
+            return fallback
+    }
+}
+
+function getDimensionAsPNG(buf: Uint8Array) {
```

Comment:
it will be better to have a comment that refer to the spec of PNG and JPEG

Reply:
A link to https://github.com/DimensionDev/Maskbook/issues/304 can reduce bytes used.

Reply:
> A link to #304 can reduce bytes used.

I mean the spec of PNG, not our Maskbook rfc